repo.json

```diff
@@ -3602,6 +3602,11 @@
           "version": "1.0.7",
           "commit": "fbb51e5df3035bdc088a13362b869091633014ef",
           "url": "https://github.com/doitintl/bigquery-grafana"
+        },
+        {
+          "version": "1.0.8",
```

Comment:
The version here, 1.0.8 is not the same as the version in the plugin.json file: 
https://github.com/doitintl/bigquery-grafana/blob/5bacd1f0c4a6bfccb0c356526806d550618e67db/dist/plugin.json#L29

The version in the plugin.json file in the dist folder needs to be updated to 1.0.8 and then this PR needs to be updated with the new commit sha.

Reply:
Thanks, Done